tests/test_http_eager.py

```diff
@@ -15,38 +15,72 @@
 """Tests for HTTP file system"""
 
 import os
-
+from tensorflow.python.platform import test
 import tensorflow as tf
 import tensorflow_io as tfio  # pylint: disable=unused-import
 
 
-def test_http_file_system():
-    """Test case for http file system"""
-    local_path = os.path.join(
-        os.path.dirname(os.path.abspath(__file__)), "test_http", "LICENSE-2.0.txt"
-    )
-    with open(local_path) as f:
-        lines = list(f)
-    local = "".join(lines)
-
-    filename = "http://www.apache.org/licenses/LICENSE-2.0.txt"
-    remote = tf.io.read_file(filename)
-
-    assert remote == local
-
-    dataset = tf.data.TextLineDataset(filename)
-    i = 0
-    for d in dataset:
-        assert d == lines[i].rstrip()
-        i += 1
-    assert i == len(lines)
-
-    remote = tf.io.gfile.GFile(filename)
-    assert remote.size() == len(local)
-    offset_start = list(range(0, len(local), 100))
-    offset_stop = offset_start[1:] + [len(local)]
-    for (start, stop) in zip(offset_start, offset_stop):
-        assert remote.read(100) == local[start:stop]
+class HTTPFSTest(test.TestCase):
```

Comment:
It looks like `tf.test.TestCase` is public API. I think we can use that directly like below?
```
class HTTPFSTest(tf.test.TestCase):
```


(The `from tensorflow.python.platform import test` is not publicly exposed endpoint so they might be subject to change in the future.

Reply:
addressed in the commit